alembic/versions/468435875f30_trt.py

```diff
@@ -0,0 +1,42 @@
+"""TRT migration
+
+Revision ID: 468435875f30
+Revises: 430a8a5f3c18
+Create Date: 2024-03-29 15:48:56.979223
+
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '468435875f30'
+down_revision = '430a8a5f3c18'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    op.execute(
+        """
+alter type "public"."followup_apis" rename to "followup_apis__old_version_to_be_dropped";
+
+create type "public"."followup_apis" as enum ('MMAAPI', 'GENERICAPI', 'SLACKAPI', 'ATLASAPI', 'GROWTHINDIAMMAAPI', 'KAITAPI', 'SEDMAPI', 'SEDMV2API', 'IOOAPI', 'IOIAPI', 'SPRATAPI', 'SINISTROAPI', 'SPECTRALAPI', 'FLOYDSAPI', 'MUSCATAPI', 'NICERAPI', 'PS1API', 'UVOTXRTAPI', 'UVOTXRTMMAAPI', 'TESSAPI', 'TRTAPI', 'WINTERAPI', 'ZTFAPI', 'ZTFMMAAPI');
```

Comment:
I know it works but... do we really need to do that to migrate??? I remember that we could simply add the new API in the correct location in the existing enum, there's a bunch of examples of that in previous migrations, seems cleaner and less risky than dropping the existing one everytime.

Reply:
@Theodlz Ooops, I forgot. Should be updated now.